client/app/common/hero/hero.js

```diff
@@ -6,6 +6,8 @@ let heroModule = angular.module('hero', [
   uiRouter
 ])
 
-.component('hero', heroComponent);
+.component('hero', heroComponent)
+  
+.name;
 
 export default heroModule;
```

Comment:
It would be cool to have a style guide along with the project. It definitely helps for teams of developers to unify. @toddmotto 's is so close to what this project already has you almost get the style guide for free as a reference which is awesome!

I'm ok with one or the other @fesor. Just let me know and I can make the necessary changes ;). Thanks for taking some time to review this.